conf/application.conf

```diff
@@ -302,6 +302,10 @@ time-machine {
     add-years = 1
 }
 
+iv {
```

Comment:
there is existing identity-verification-frontend key in the config, we might don't need add anything new on this occasion; just re-use what we have so far